nl.lproj/Localizable.strings

```diff
@@ -9,7 +9,7 @@
 "SHORTCUT_PLAY" = "Speel";
 "SHORTCUT_PAUSE" = "Pauzeer";
 
-"TOOLBAR_NOTHING_PLAYING" = "Niets aan het spelen";
+"TOOLBAR_NOTHING_PLAYING" = "Er wordt niks afgespeeld";
```

Comment:
I suggest using “niets” instead of “niks”, as the latter is more informal.

Reply:
I agree! I changed it.